packages/components/src/DateTimePickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -57,6 +87,20 @@ class DateTimePicker extends React.Component {
 		this.setState({ isDateTimeView });
 	}
 
+	trySubmit() {
```

Comment:
Or maybe the other way, remove the other method and take it's code back here ?!
The `try` has it's importance, `submit()` don't tell us it could not submit on some cases

Reply:
yes, one or the other, the problem here is to have those 2 methods

Reply:
I don't see for what reason it's a problem ?! But I can do the change.

Reply:
✔